adafruit_vl53l1x.py

```diff
@@ -31,7 +31,7 @@
 from adafruit_bus_device import i2c_device
 from micropython import const
 
-__version__ = "0.0.0+auto.0"
+__version__ = "1.1.14"
```

Comment:
Please change the version back to `0.0.0+auto.0` the real values get filled in automatically during the release process.